clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -1460,8 +1456,8 @@ void SYCLToolChain::TranslateBackendTargetArgs(
     // have implied the setting.
```

Comment:
Should this comment be updated to include spirv64?

Reply:
I made a full pass of comments and updated where necessary. Thanks for feedback.